recipes/mysql_master.rb

```diff
@@ -18,49 +18,50 @@
 # limitations under the License.
 #
 
-# set repl password
-::Chef::Recipe.send(:include, Opscode::OpenSSL::Password)
-node.set_unless['mysql-multi']['server_repl_password'] = secure_password
-
+include_recipe 'apt' if node.platform_family?('debian')
```

Comment:
From the readme for the `apt` cookbook, 'Including the default recipe on a node that does not support apt (such as Windows) results in a noop.' I'm not sure you need the check on node.platform_family? here (or in the other recipes).